src/commandHandlers/issue.ts

```diff
@@ -33,28 +28,47 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         // await repository name
         let repository = await message.channel.awaitMessages((m: Message) => m.author.id === message.author.id, { max: 1, time: 10000, errors: ['time'] }).then(collected => {
             const res = collected.first();
-            if(res) return res.content;
-            else return 'EddieBot';
+            if(res) {
+                return res.content;
+            }
+            return 'EddieBot';
         }).catch(() => { throw new Error('You took too long to reply (`10 seconds`). Try again.') });
 
         // query max length is 256 chars
         repository = encodeURI(repository);
-        if(repository.length > 256) repository.substr(0, 256);
+        if(repository.length > 256) {
+            repository.substr(0, 256);
+        }
```

Comment:
what do you think of moving this comment into an explicit variable?
```suggestion
        const MAX_QUERY_LENGTH = 256;
        repository = encodeURI(repository);
        if (repository.length > MAX_QUERY_LENGTH) {
            repository.substr(0, MAX_QUERY_LENGTH);
        }
```

Reply:
Didn't think of that, thanks for reminding me